src/views/DogDetail.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { useParams } from 'react-router-dom';
+import { deleteDog, getDogById } from '../services/DogList';
+import Navbar from './Navbar';
+
+export default function DogDetail() {
+  const params = useParams();
+  const [dog, setDog] = useState({});
+  const [loading, setLoading] = useState(true);
+
+  const history = useHistory();
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const data = await getDogById(params.id);
+      setDog(data);
+      setLoading(false);
+    };
+    fetchData();
+  });
+
+  if (loading) return <div className="loader"> Loading Dog</div>;
+
+  return (
+    <div>
+      <Navbar />
+      <p></p>
+      <p></p>
+      <h2>{dog.name}</h2>
+      <h2>{dog.breed}</h2>
+      <h2>{dog.age}</h2>
+      <h2>{dog.bio}</h2>
+      <h2>{dog.image}</h2>
```

Comment:
We can render a img element here and place the src as the dog.image to render the image to the page.